tests/minimise/in17.re

```diff
@@ -0,0 +1 @@
+[aBx]*a
```

Comment:
We already have this as `in0.re`

Reply:
I'm not sure how this ended up here; removed.

Reply:
It's getting to be a famous test case now :)